packages/shared/src/components/GoToDevCardButton.tsx

```diff
@@ -1,30 +1,37 @@
 import React, { ReactElement, ReactNode } from 'react';
-import classNames from 'classnames';
 import useGoToDevCardButton from '../hooks/useGoToDevCardButton';
 import { Button } from './buttons/Button';
+import { SimpleTooltip } from './tooltips/SimpleTooltip';
 
 export type GoToDevCardButtonProps = {
   origin: string;
   children?: ReactNode;
   className?: string;
+  isLocked?: boolean;
 };
 
 export default function GoToDevCardButton({
   origin,
   children,
   className,
+  isLocked,
 }: GoToDevCardButtonProps): ReactElement {
   const [onGenerateDevCardClick] = useGoToDevCardButton(origin);
 
   return (
-    <Button
-      className={classNames('btn-primary', className)}
-      tag="a"
-      buttonSize="small"
-      href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}devcard`}
-      onClick={onGenerateDevCardClick}
-    >
-      {children}
-    </Button>
+    <SimpleTooltip content="Sign up to Unlock">
```

Comment:
Now it will always show this text right?
Also for signed in users?

Reply:
Oh, wait. Forgot to revert as I was debugging earlier. Nice spot.